packages/core/src/types.ts

```diff
@@ -13,3 +13,51 @@ export type CommonProps = {
   /** Styles */
   style?: CSSProperties;
 };
+
+export type Spacing =
```

Comment:
I think we should take a general decision on whether or not we want to use enums for cases like this or if we only want to support strings. If we want to do enums the value should also be able to be passed as e.g. `Spacing.2Xs`

Reply:
We didn't use enums before and I'm not sure if we want to start this. Almost all popular React component libraries are using string literals and people don't have problems with that.

Reply:
It'd probably be mostly for our internal use. I don't think it should be either/or, but an addition. I like how well enums work as a user of a library but I'm fine with strings too.